spec/features/users/user_can_search_rdv_on_rdv_mairie_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+describe "User can search rdv on rdv mairie" do
+  let(:now) { Time.zone.parse("2021-12-13 8:00") }
+
+  def json_response
+    JSON.parse(page.html)
+  end
+
+  before do
+    default_url_options[:host] = "http://www.rdv-mairie-test.localhost"
+  end
+
+  around do |example|
+    previous_auth_token = ENV["ANTS_API_AUTH_TOKEN"]
+    ENV["ANTS_API_AUTH_TOKEN"] = ""
+
+    example.run
+
+    ENV["ANTS_API_AUTH_TOKEN"] = previous_auth_token
+  end
+
+  before do
+    travel_to(now)
+    create(:plage_ouverture, :no_recurrence, first_day: now, motifs: [motif], lieu: lieu, organisation: organisation, start_time: Tod::TimeOfDay(9), end_time: Tod::TimeOfDay.new(10))
+  end
+
+  describe "default" do
+    let!(:territory95) { create(:territory, departement_number: "95") }
+    let!(:organisation) { create(:organisation, :with_contact, territory: territory95, verticale: :rdv_mairie) }
+    let(:service) { create(:service) }
+    let!(:motif) { create(:motif, name: "Passeport", bookable_publicly: true, organisation: organisation, restriction_for_rdv: nil, service: service) }
+    let!(:lieu) { create(:lieu, organisation: organisation, name: "Mairie de Sannois", address: "15 Place du Général Leclerc, Sannois, 95110") }
+
+    it "default" do
+      lieux_ids = fetch_lieux_ids_from_ants
+      creneaux_url = fetch_time_slots_from_ants(lieux_ids)
```

Comment:
idem ici, et pour une question de naming, là on ne va pas chercher les choses depuis l'ants, mais plutôt depuis l'api qu'on met à disposition pour l'ants (nous faisons partie de l'anct, pas de l'ants).

Reply:
Done in 9bdfe74bee86a5e7a4dddb0f0d49b53cadd28fbf